src/eventListeners/touchEventListeners.js

```diff
@@ -23,7 +23,7 @@ let lastScale = 1.0,
 const pressDelay = 700,
   pressMaxDistance = 5;
 
-const toolType = 'touchInput';
+const toolName = 'touchInput';
```

Comment:
I'm not convinced this is supposed to be `toolName`. "touchInput" appears to be specific to the interaction that's happening here, this is taking place in the `touchEventListener`

Reply:
Renamed to `inputName` as it's not a tool.
Also added some `TODO`s as it's calling `setToolOptions` to store `hammer.js` options.